### PR TITLE
Make the number of droplets to keep in the v2 API obey the same count as v3

### PR DESCRIPTION
**Pull request by mook-as**
This lets v2 droplets hang around in the same way the v3 ones do.

Also fixes a copy-and-paste error in v3 droplet/package expiration where the two used each other's configuration.

### CODE REVIEW

File: lib/cloud_controller/bits_expiration.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ def initialize(input_config=Config.config)
       config = {}
       config[:packages] = input_config[:packages] || {}
       config[:droplets] = input_config[:droplets] || {}
-      @droplets_storage_count = config[:packages][:max_valid_packages_stored] || 5
-      @packages_storage_count = config[:droplets][:max_staged_droplets_stored] || 5
+      @packages_storage_count = config[:packages][:max_valid_packages_stored] || 5
```
Comment (jandubois) [inline lib/cloud_controller/bits_expiration.rb:7]: Isn't this a separate bug fix that should be in its own commit?

Comment (mook-as) [inline lib/cloud_controller/bits_expiration.rb:7]: Fixed in c16e21389433207fec357015de312054b4ac5771

**Comment by mook-as** [comment]
Looks good enough, going upstream.
